src/main/java/org/jitsi/jicofo/JitsiMeetConferenceImpl.java

```diff
@@ -772,41 +884,37 @@ public void destroy(String reason)
     }
 
     /**
-     * Expires the conference on the bridge and other stuff related to it.
+     * Disposes of this conference. Expires all allocated COLIBRI conferences.
+     *
+     * Does not terminate jingle sessions with its participants (why???).
+     *
      */
     private void disposeConference()
     {
-        synchronized (colibriConfSyncRoot)
+        if (recording != null)
         {
-            // We dispose the recorder here as the recording session is usually
-            // bound to Colibri conference instance which will be invalid once we
+            recording.dispose();
+            recording = null;
+        }
 
-            // dispose/expire the conference on the bridge
-            if (recording != null)
-            {
-                recording.dispose();
-                recording = null;
-            }
+        // If the conference is being disposed the timeout is not needed
+        // anymore
+        cancelSinglePeerTimeout();
 
-            // If the conference is being disposed the timeout is not needed
-            // anymore
-            cancelSinglePeerTimeout();
-
-            if (colibriConference != null)
+        synchronized (bridges)
+        {
+            for (BridgeDesc bridgeDesc : bridges)
             {
-                // We will not expire channels if the bridge is faulty or
-                // when our connection is down
-                if (!bridgeHasFailed && protocolProviderHandler.isRegistered())
-                {
-                    colibriConference.expireConference();
-                }
-                else
-                {
-                    colibriConference.dispose();
-                }
-                colibriConference = null;
+                // No need to expire channels, just expire the whole colibri
+                // conference.
+                // bridgeDesc.terminateAll();
+                bridgeDesc.dispose();
+                bridges.remove(bridgeDesc);
             }
         }
+
+        // TODO: what about removing the participants and ending their jingle
```

Comment:
Yes, the colibri channels are still expired. But we are still left with Participant instances and their jingle sessions. Normally I would expect dispose() to take care of these, but it's not a prio so I only added a comment for now.